src/aesthetics.jl

```diff
@@ -1,24 +1,24 @@
 const NumericalOrCategoricalAesthetic =
-    Union{(Void), Vector, DataArray, IndirectArray}
+    Union{Nothing, Vector, Array, IndirectArray}
```

Comment:
It sounds weird to have both `Vector` and `Array`. Same below.

Reply:
I agree. In the Gadfly aesthetics, only `z` can be a 1 or 2-dimensional array, everything else is 1-dimensional (i.e. `Vector`).